upstream/run_rust_keylime_tests/test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+#[ -n "${RUST_KEYLIME_UPSTREAM_URL}" ] || RUST_KEYLIME_UPSTREAM_URL="https://github.com/keylime/rust-keylime.git"
+#[ -n "${RUST_KEYLIME_UPSTREAM_BRANCH}" ] || RUST_KEYLIME_UPSTREAM_BRANCH="master"
+
+UPLOAD_URL=https://transfer.sh
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        if [ -d /var/tmp/rust-keylime_sources ]; then
+            rlLogInfo "Missing upstream rust keylime sources."
+        else
+            rlDie "Upstream keylime sources must be already downloaded."
+        fi
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            # make sure tpm2-abrmd is running
+            rlServiceStart tpm2-abrmd
+            sleep 5
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        fi
+        rlRun "pushd /var/tmp/rust-keylime_sources"
+        rlRun "cargo install cargo-tarpaulin"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Run unit tests"
+        #run cargo tarpaulin code coverage
```

Comment:
I think we should make all the code coverage related commands conditional.. so it can also work without a code coverage measurement. For troubleshooting we may not want to measure code coverage.

Reply:
I tried to make conditional, so please take a look if it's sufficient.